com.github.hugolabe.Wike.json

```diff
@@ -0,0 +1,44 @@
+{
+  "app-id" : "com.github.hugolabe.Wike",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.38",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "wike",
+  "finish-args" : [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
```

Comment:
You probably want `--device=dri` as well

Reply:
Ok. I add it.